docs/ref/reference.rst

```diff
@@ -321,6 +328,11 @@ When `--nodename` is omitted, it is computed as above (see
 :ref:`_pg_autoctl_create_monitor`), with the difference that step 1 uses the
 monitor IP and port rather than the public service 8.8.8.8:53.
 
+The ``--auth`` option allows setting up authentication method to be used when
+monitor node makes a connection to data node with `pgautofailover_monitor` user.
+If this option to be used, please make sure password is manually set on the data
```

Comment:
Same as before, “if this option is used”. Could also be written “When using this option”.

Reply:
fixed